scripts/test_version_tags.sh

```
@@ -0,0 +1,77 @@
+
+set -e
+
+echo "🔍 Testing version tag creation logic..."
+
+# Extract version from mix.exs (same as workflow)
+VERSION=$(grep -E '^\s+version:' mix.exs | head -1 | grep -o '"[^"]*"' | tr -d '"')
+echo "📋 Found version in mix.exs: $VERSION"
+
+# Create tag names
+TAG="v$VERSION"
+IFS='.' read -r MAJOR MINOR PATCH <<< "$VERSION"
```

Shellcheck (0.10.0)

[warning] PATCH appears unused. Verify use (or export if used externally).

(SC2034)